conf/documentation.conf

```diff
@@ -1315,6 +1329,16 @@ description=<<EOT
 Disable Active-Directory domain configuration from within PacketFence
 EOT
 
+[advanced.ntlm_redis_cache]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Enables a Redis driven cache for NTLM authentication. 
+In order for this to work, you need to setup proper NT hash syncronization between your PacketFence server and your AD.
+Refer to the Administration guide for more details.
+Applying this requires a restart of radiusd.
```

Comment:
there's no "radiusd" - say "radius.authentication" - all of them? just be more precise.